en:
  errors:
    unexpected_value: "Unexpected %{variable_name} `%{value}`"
    challenges:
      update:
        invalid_update_type: "Invalid update type %{update_type}"
